fdbserver/VersionedBTree.actor.cpp

```diff
@@ -1030,6 +1030,22 @@ class ObjectCache : NonCopyable {
 		return nullptr;
 	}
 
+	// Try to evict the item at index from cache
+	// Returns true if item is evicted or was not present in cache
```

Comment:
Looks like it's returning false for the later case?

Reply:
Good catch!  I'll fix this.